parser.y

```diff
@@ -3853,13 +3857,37 @@ SumExpr:
 	{
 		$$ = &ast.AggregateFuncExpr{F: $1, Args: []ast.ExprNode{$4}, Distinct: $3.(bool)}
 	}
-|	builtinStddevPop '(' BuggyDefaultFalseDistinctOpt Expression ')'
+|	builtinStddevPop '(' Expression ')'
```

Comment:
We'd better support `distinct` clause in variance aggregate function.

Reply:
@mccxj After discussion, we decide to support `distinct` for every aggregate functions.